cmp/compare.go

```diff
@@ -6,6 +6,9 @@
 //
 // This package is intended to be a more powerful and safer alternative to
 // reflect.DeepEqual for comparing whether two values are semantically equal.
+// It is intended to only be used in tests, as cmp may panic if it cannot
+// compare the values. It's propensity towards panicking means that it is
```

Comment:
Its

Reply:
Kept it in untracted form for the first occurrence. Used contracted form for later occurences.

Reply:
Oh, you were probably referring to the "it's", which is incorrect grammar. Sigh, the inability to highlight a specific word in GitHub PR is confusing.